### PR TITLE
Add more unit tests

### PR DESCRIPTION
**Pull request by glonlas**
Just for the fun, more unit tests more coverage \o/

### CODE REVIEW

File: freqtrade/tests/test_main.py [hunk 1]
```diff
@@ -213,6 +214,30 @@ def test_handle_trade(default_conf, limit_buy_order, limit_sell_order, mocker):
...
+    default_conf.update({'experimental': {'use_sell_signal': True}})
+    mocker.patch.dict('freqtrade.main._CONF', default_conf)
+
+    mocker.patch('freqtrade.main.get_signal', side_effect=lambda s, t: True)
+    mocker.patch.multiple('freqtrade.rpc', init=MagicMock(), send_msg=MagicMock())
+    mocker.patch.multiple('freqtrade.main.exchange',
+                          validate_pairs=MagicMock(),
+                          get_ticker=ticker,
+                          buy=MagicMock(return_value='mocked_limit_buy'))
+    mocker.patch('freqtrade.main.min_roi_reached', MagicMock(return_value=True))
```
Comment (vertti) [inline freqtrade/tests/test_main.py]: mocker.patch uses MagicMock by default so you can simplify that with just 

```
mocker.patch('freqtrade.main.min_roi_reached', return_value=True)
```

Comment (glonlas) [reply freqtrade/tests/test_main.py]: Thanks for the tips. I will do it.

**Comment by glonlas** [comment]
@vertti your comment is fixed. Waiting for the tests to pass.

**Comment by vertti** [comment]
Looks good to me.

**Pull request merged**